src/components/site/SummaryList.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+const SummaryList = props => {
+  const {
+    iconMarker,
+    iconClock,
+    iconEnvelop,
+    iconMobile,
+    iconCircle,
+    iconUser
+  } = props;
+  const iconCssClassesMain = classnames("side-list-ico", iconMarker);
+  const iconCssClassesHours = classnames("side-list-ico", iconClock);
+  const iconCssClassesEmail = classnames("side-list-ico", iconEnvelop);
+  const iconCssClassesPhone = classnames("side-list-ico", iconMobile);
+  const iconCssClassesBaltcogo = classnames("side-list-ico", iconCircle);
+  const iconCssClassesDirector = classnames("side-list-ico", iconUser);
+  return (
+    <React.Fragment>
+      <div className="dg_summary-list">
+        <div className="col-md-12 bg-light px-4 py-4">
```

Comment:
we want to control the padding and margin through css not these helper classes when possible. The `p-` and `m-` classes respectively.

Reply:
`bg-light` should be removed and a background color should be added to `dg_summary-list`

Reply:
done